source/buildings/university.md

```diff
@@ -55,27 +55,287 @@ Page 2 of the GUI will show you the options for each research tree.
+| Hamlet | Outpost | House at least level 5; 128 steak | Lets you have over 50 colonists |
+| Studious | Diligent | Library at least level 3; 12 books | Increases the rate colonists level up by 10% |
+| Nimble | Rails | Town Hall at least level 3; 1 rabbit foot | Increases colonists' speed by 5% |
+| Lifesaver | First Aid II | Town Hall at least level 3; 32 hay bales | Increases colonists' health by 3 hearts |
+| Festival | Circus | Restaurant at least level 3; 9 cakes | Increases colonists' happiness by 10% |
+| Nightowl | Circus | Town Hall at least level 3; 25 golden carrots | Colonists' bedtime is pushed back by 1 in-game hour |
+| Gorger | Gourmand | Restaurant at least level 3; 64 cookies | The saturation of every food colonists eat goes up by 25% |
+| Bookworm | More Books | School at least level 3; 6 bookshelves | Increases the effectiveness of Teachers by 10% |
+| Hormones | Nurture | School at least level 3; 64 cooked chicken | Decreases the time it takes children to grow up by 10% |
+
+### Fourth Column
+
+| Name | Prerequisites | Requirements | Effects |
+| ---- | ------------- | ------------ | ------- |
+| Bandages | Healing Cream | Library at least level 4; 16 golden carrots | Every tick that a colonist is healing, they heal by 50% more |
+| Vitality | Resilience | Restaurant at least level 4; 16 golden apples | Normally colonists can only heal if they have full saturation. With this research, they can heal if they have a shank and a half less |
+| Village | Hamlet | Town Hall at least level 4; 256 steak | Lets you have over 75 colonists |
+| Scholarly | Studious | Library at least level 4; 24 books | Increases the rate colonists level up by 25% |
+| Agile | Nimble | Town Hall at least level 4; 10 rabbit feet | Increases colonists' speed by 10% |
+| Lifesaver II | Lifesaver | Town Hall at least level 4; 64 hay bales | Increases colonists' health by 4 hearts |
+| Spectacle | Festival | Restaurant at least level 4; 18 cakes | Increases colonists' happiness by 15% |
+| Nightowl II | Festival | 75 golden carrots | Colonists' bedtime is pushed back by 2 in-game hours (3 hours total if you've also researched Nightowl) |
+| Stuffer | Gorger | Restaurant at least level 4; 128 cookies | The saturation of every food colonists eat goes up by 50% |
+| Bachelor | Bookworm | Library at least level 3; 12 bookshelves | Increases the effectiveness of Teachers by 25% |
+| Puberty | Hormones | Library at least level 3; 128 cooked chicken | Decreases the time it takes children to grow up by 25% |
+
+### Fifth Column
+
+| Name | Prerequisites | Requirements | Effects |
+| ---- | ------------- | ------------ | ------- |
+| Compress | Bandages | Library level 5; 32 golden carrots | Every tick that a colonist is healing, they heal by 100% more |
+| Fortitude | Vitality | Restaurant level 5; 32 golden apples | Normally colonists can only heal if they have full saturation. With this research, they can heal if they have 2 shanks less |
```

Comment:
Some people might not immediately know what shanks refers to. Not sure though. (and the one further up and further down)

Reply:
```suggestion
| Fortitude | Vitality | Restaurant level 5; 32 golden apples | Colonists can heal with only 80% saturation |
```
It should be satiation really, but I'd rather be consistent.